packages/tools/locales/hr_HR.po

```diff
@@ -3923,254 +3955,3 @@ msgstr "Označava zadatak završenim."
-#~ "postojeću bilježnicu iz izbornika."
-
-#~ msgid ""
-#~ "You currently have no notebook. Create one by clicking on (+) button."
-#~ msgstr "Trenutno nemaš nijednu bilježnicu. Stvori novu klikom na (+) gumb."
-
-#~ msgid "Welcome"
-#~ msgstr "Dobro došli"
-
-#~ msgid "Separate each tag by a comma."
-#~ msgstr "Odvoji oznake zarezom."
-
-#~ msgid ""
-#~ "The path to synchronise with when file system synchronisation is enabled. "
-#~ "See `sync.target`."
-#~ msgstr ""
-#~ "Putanja do direktorija za sinkronizaciju u slučaju kad je sinkronizacija "
-#~ "sa datotečnim sustavom omogućena. Vidi `sync.target`."
-
-#, fuzzy
-#~ msgid "State: %s."
-#~ msgstr "Stanje: \"%s\"."
-
-#~ msgid "A notebook with this title already exists: \"%s\""
-#~ msgstr "Bilježnica s ovim naslovom već postoji: \"%s\""
-
-#~ msgid "Searches"
-#~ msgstr "Pretraživanja"
-
-#~ msgid "Imports an Evernote notebook file (.enex file)."
-#~ msgstr "Uvozi Evernote bilježnicu (.enex datoteku)."
-
-#~ msgid ""
-#~ "File \"%s\" will be imported into existing notebook \"%s\". Continue?"
-#~ msgstr ""
-#~ "Datoteka \"%s\" će biti uvezena u postojeću bilježnicu \"%s\". Nastavi?"
-
-#~ msgid ""
-#~ "New notebook \"%s\" will be created and file \"%s\" will be imported into "
-#~ "it. Continue?"
-#~ msgstr ""
-#~ "Nova bilježnica \"%s\" će biti stvorena i datoteka \"%s\" će biti uvezena "
-#~ "u nju. Nastavi?"
-
-#~ msgid "Import Evernote notes"
-#~ msgstr "Uvezi Evernote bilješke"
-
-#~ msgid "Give focus to next pane"
-#~ msgstr "Fokusiraj sljedeće okno"
-
-#~ msgid "Give focus to previous pane"
-#~ msgstr "Fokusiraj prethodno okno"
-
-#~ msgid "Exit command line mode"
-#~ msgstr "Napusti naredbeni redak"
-
-#~ msgid "Cancel the current command."
-#~ msgstr "Prekini trenutnu naredbu."
-
-#~ msgid "Set a to-do as completed / not completed"
-#~ msgstr "Postavi zadatak kao završen/nezavršen"
-
-#, fuzzy
-#~ msgid "[t]oggle [c]onsole between maximized/minimized/hidden/visible."
-#~ msgstr "[t]oggle [c]onsole between maximized/minimized/hidden/visible."
-
-#, fuzzy
-#~ msgid "[t]oggle note [m]etadata."
-#~ msgstr "[t]oggle note [m]etadata."
-
-#, fuzzy
-#~ msgid "[M]ake a new [n]ote"
-#~ msgstr "[M]ake a new [n]ote"
-
-#, fuzzy
-#~ msgid "[M]ake a new [t]odo"
-#~ msgstr "[M]ake a new [t]odo"
-
-#, fuzzy
-#~ msgid "[M]ake a new note[b]ook"
-#~ msgstr "[M]ake a new note[b]ook"
-
-#, fuzzy
-#~ msgid "Copy ([Y]ank) the [n]ote to a notebook."
-#~ msgstr "Copy ([Y]ank) the [n]ote to a notebook."
-
-#~ msgid "Move the note to a notebook."
-#~ msgstr "Premjesti bilješku u bilježnicu."
-
-#~ msgid ""
-#~ "The target to synchronise to. If synchronising with the file system, set "
-#~ "`sync.2.path` to specify the target directory."
-#~ msgstr ""
-#~ "Meta sinkronizacije. U slučaju sinkroniziranja s vlastitim datotečnim "
-#~ "sustavom, postavi `sync.2.path` na ciljani direktorij."
-
-#~ msgid "To-do title:"
-#~ msgstr "Naslov zadatka:"
-
-#~ msgid "\"%s\": \"%s\""
-#~ msgstr "\"%s\": \"%s\""
```

Comment:
Please do not delete these. We keep old translations around. The reason being, aource strings might be removed bu6t latre added again. In that case, we already have a proper translation (if it was translated in the past).